kclvm/runner/src/command.rs

```diff
@@ -152,6 +63,51 @@ impl Command {
         path.to_str().unwrap().to_string()
     }
 
+    /// Add args for cc.
+    pub(crate) fn add_args(
+        &self,
+        libs: &[String],
+        lib_path: String,
+        cmd: &mut std::process::Command,
+    ) {
+        #[cfg(not(target_os = "windows"))]
+        self.unix_args(libs, cmd);
+
+        #[cfg(target_os = "windows")]
+        self.msvc_win_args(libs, lib_path, cmd);
+    }
+
+    // Add args for cc on unix os.
+    pub(crate) fn unix_args(&self, libs: &[String], cmd: &mut std::process::Command) {
+        cmd.args(libs)
+            .arg(&format!("-Wl,-rpath,{}/lib", self.executable_root))
+            .arg(&format!("-L{}/lib", self.executable_root))
+            .arg(&format!("-I{}/include", self.executable_root))
+            .arg("-lkclvm");
+    }
+
+    // Add args for cc on windows os.
+    pub(crate) fn msvc_win_args(
+        &self,
+        libs: &[String],
+        lib_path: String,
+        cmd: &mut std::process::Command,
+    ) {
+        cmd.args(libs)
+            .arg("kclvm.lib")
+            .arg("/link")
+            .arg("/ENTRY:kclvm_main")
+            .arg("/NOLOGO")
+            .arg(format!(r#"/LIBPATH:"{}\_output""#, self.executable_root))
```

Comment:
What does the path `{executable_root}\_output` mean? Do the windows building artifact path? Will this path appear in the package for users?

Reply:
> 

Put all kcl windows artifacts into the `bin` directory.

Reply:
fixed